docs/user-guide/configure.md

```diff
@@ -1,29 +1,20 @@
-# Configuration
+# Configuring
 
-Stylelint _expects a configuration object_.
+Stylelint expects a configuration object, and looks for one in a:
 
-Stylelint uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) to find and load your configuration object. Starting from the current working directory, it looks for the following possible sources:
+- `stylelint` property in `package.json`
+- `.stylelintrc{.js,.json,.yaml,.yml}` file
```

Comment:
[suggest] What about splitting `.stylelintrc` and `.styelintrc.*` for clarity, and adding a `cjs` extension?

```suggestion
- `.stylelintrc` file
- `.stylelintrc.{cjs,js,json,yaml,yml}` file
```

Reply:
@ybiquitous do we support `mjs` as well?

Reply:
cosimiconfig doesn't support `mjs` yet, see https://github.com/davidtheclark/cosmiconfig/issues/224